content/hi/docs/reference/glossary/cluster-operator.md

```diff
@@ -0,0 +1,22 @@
+---
+title: क्लस्टर ऑपरेटर
+id: cluster-operator
+date: 2018-04-12
+full_link:
+short_description: >
+  एक व्यक्ति जो क्लस्टर को कॉन्फ़िगर, नियंत्रित और मॉनिटर करता है।
+
+aka:
+tags:
+  - user-type
+---
+
+एक व्यक्ति जो क्लस्टर को कॉन्फ़िगर, नियंत्रित और मॉनिटर करता है।
+
+<!--more-->
+
+उनकी प्राथमिक जिम्मेदारी एक क्लस्टर को ऊपर रखना और चलाना है, जिसमें आवधिक रखरखाव गतिविधियां या उन्नयन शामिल हो सकते हैं।<br>
+
+{{< note >}}
+क्लस्टर ऑपरेटर [ऑपरेटर पैटर्न](https://www.openshift.com/learn/topics/operators) से भिन्न होते हैं जो कुबेरनेट्स एपीआई का विस्तार करता है।
```

Comment:
If we're referring to operators in plural, then I think the verb should follow the noun?
```suggestion
क्लस्टर ऑपरेटर [ऑपरेटर पैटर्न](https://www.openshift.com/learn/topics/operators) से भिन्न होते हैं जो कुबेरनेट्स एपीआई का विस्तार करते है।
```